src/vconsole/vconsole-setup.c

```diff
@@ -76,6 +76,10 @@ static int toggle_utf8(const char *name, int fd, bool utf8) {
 
         assert(name);
 
+        r = vt_verify_kbmode(fd);
+        if (r < 0) {
+                return log_debug_errno(r, "Virtual console %s is not in K_XLATE or K_UNICODE: %m", src_vc);
```

Comment:
not valid C. The { should not be there

Reply:
Also, I figure we eat up the error in this case, given that we probably either log about this loudly (but we shouldn't in this case) or eat it up. Hence please rework to:

```c
if (r < 0) {
        log_debug_errno(r, "Virtual console %s is not in K_XLATE or K_UNICODE mode, not changing UTF-8 mode: %m");
        return 0;
}
```